app/routes/legal-status.js

```diff
@@ -15,29 +15,62 @@ function createModel (errorMessage) {
         }
       },
       items: [
+        {
+          value: 'Limited Company',
+          text: 'Limited Company'
+        },
+        {
+          value: 'Partnership',
+          text: 'Partnership'
+        },
+        {
+          value: 'Sole trader',
+          text: 'Sole trader'
+        },
+        {
+          value: 'Limited Liability Company',
+          text: 'Limited Liability Company'
+        },
+        {
+          value: 'Trust',
+          text: 'Trust'
+        },
         {
           value: 'Charity',
           text: 'Charity'
         },
         {
-          value: 'Community interest organisation',
-          text: 'Community interest organisation'
+          value: 'Community Interest Company',
+          text: 'Community Interest Company'
         },
         {
-          value: '...',
-          text: '...'
+          value: 'Public Organisation',
+          text: 'Public Organisation'
+        },
+        {
+          value: 'Other',
+          text: 'Other'
         }
       ],
       ...(errorMessage ? { errorMessage: { text: errorMessage } } : {})
     }
   }
 }
 
-function createModelNotEligible () {
+function createModelNotEligible() {
   return {
     backLink: '/legal-status',
     sentences: [
-      'This is only for projects for an arable or horticultural farming businesses.'
+      'This is only open to a business with a legal status of:',
+      '\n\u2022 Charity',
```

Comment:
The rendering of these bullet points will probably get questioned as they aren't styled with the govuk-frontend styling.

Have you considered instead of passing through an array of sentences to render in the template, just passing through a block of mark-up?

So pass through something like:

`messageContent: '<p class="govuk-body">This is only open to a business with a legal status of: <ul class="govuk-list govuk-list--bullet"><li>Charity</li></ul></p>'`

Then in nunjucks you can just render it with `{{ messageContent | safe}}`